0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,129 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection extension protocol
```

Comment:
`Collection` is capitalized as a part of "Data Collection" proper noun. I consider `extension`, `service` and `protocol` common nouns because a) we've got too many of them b) often times they aren't even a part of the acronym.
Kaa documentation guidelines do not apply to RFCs, but in areas where RFC guidelines fall short of defining anything related to style (like capitalization for proprietary terms), it better be inherited from the nearest context. In our case, it's Kaa documentation. Because what RFC guidelines do define clearly is that capitalization must be consistent accross the documentation. See para. 3.4: https://www.rfc-editor.org/rfc/rfc7322.txt